contributor/outline.md

```diff
@@ -1,64 +1,141 @@
+  The development, review, and deployment process for your contribution will likely take longer than similar processes on your own team.
+  Keep that in mind and plan to start early enough so that your work is available for you to leverage at the time you need it.
+    - If the host team is in a different time zone then keep that in mind and plan for it when interacting with them.
+    - For larger contributions, get agreement from the host team on the user/technical design of the contribution _before_ submitting a pull request.
+  This type of high-level, up-front agreeemment early will save time in rework or rejection of your pull request down the road.
+    - Understand how the host team would like you to reach out for help and questions while you're working.
+  - Development
+    - If you get stuck, try to unblock yourself by reading the project documentation or looking at other parts of the code in the project.
+    - After trying the above if you're still stuck then reach out to the host team for help.
+    It doesn't do either side any good for you to spin forever on a problem that could be resolved in a few-minute conversation with someone more knowledgable about the project.
+    It's OK to ask for help.
+    You aren't expected to be as knowledgable and proficient as you might be on the full-time projects that your team owns.
+    - Match the existing coding style, indentation, etc, even if it's not what you would normally do.
+    A guest contribution for a feature is not the time to introduce a new way of structuring or formatting project code.
+  - Submitting the pull request
+    - Run the project build and automated validation before opening a pull request.
+    - Test out your contribution before opening the pull request.
+      - Ideally you can create a modified build of the shared project with your changes and try it out in your own project that consumes it.
+    - Ensure that your pull request includes any documentation updates that are relevant to your changes.
+    - Be sure that your pull request includes just the relevant changes for the issue you're completing.
+    Making pull requests too broad and large makes them more difficult to review and longer before they're accepted.
+  
+## Benefits of Contribution
+
+  - Personally
+    - Puts your development time into a place where others are also working.
+    It's not just up to you to maintain this area.
+    - Get help and support from the host team in implementing your changes.
+    - Your reputation and impact expands to outside your team.
+    - Learn from engineers on the host team and their coding and working styles.
+    This extra learning comes as part of your day job - no stretch or overtime assignment needed.
+    - Varied, fun, and personally fulfilling.
+  - Team
+    - You can get improvements you need in other projects on a timeline that you control.
+    - High influence into the development, progression, and ecosystem of projects that you depend on.
```

Comment:
Whether or not the influence is high depends on your rank in the host project. It will be rather low in the beginning and might grow, but so does the commitment you need to make. I would rephrase this to a statement saying that this model is an _opportunity to influence_.

Reply:
Good point.